install_config/redeploying_certificates.adoc

```diff
@@ -688,6 +682,18 @@ $ oc create secret tls router-certs --cert=router.pem \ <1>
 <1> *_router.pem_* is the file that contains the concatenation of the
 certificates that you generated.    
 
+. Redeploy the router:
++
+----
+$ oc rollout latest dc/router
+----
+
```

Comment:
Need to add a `+` here so that the ordered list doesn't break from the text below.

Reply:
Thank you for your review ! I've done as mentioned.